package.json

```diff
@@ -34,37 +34,37 @@
     "node": "^12 || ^14 || ^16"
   },
   "dependencies": {
-    "@types/node": "^16.0.0",
-    "extend": "^3.0.2",
-    "ibm-cloud-sdk-core": "~2.14.0"
+    "@types/node": "16.7.10",
+    "extend": "3.0.2",
+    "ibm-cloud-sdk-core": "2.14.1"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@types/jest": "^27.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.15.1",
-    "@typescript-eslint/eslint-plugin-tslint": "^4.15.1",
-    "@typescript-eslint/parser": "^4.15.1",
-    "axios": "^0.21.0",
-    "dotenv": "^10.0.0",
-    "eslint": "^7.20.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-config-prettier": "^8.0.0",
-    "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-header": "^3.0.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "jest": "^26.1.0",
-    "jest-junit": "^12.0.0",
-    "prettier": "^2.1.2",
-    "semantic-release": "^17.2.3",
-    "sinon": "^11.0.0",
-    "tough-cookie": "^4.0.0",
-    "ts-jest": "^26.1.2",
-    "tsc-publish": "^0.5.2",
-    "typedoc": "^0.21.0",
-    "typescript": "^4.1.5"
+    "@semantic-release/changelog": "5.0.1",
+    "@semantic-release/git": "9.0.0",
+    "@types/jest": "27.0.1",
+    "@typescript-eslint/eslint-plugin": "4.30.0",
+    "@typescript-eslint/eslint-plugin-tslint": "4.30.0",
+    "@typescript-eslint/parser": "4.30.0",
+    "axios": "0.21.1",
+    "dotenv": "10.0.0",
+    "eslint": "7.32.0",
+    "eslint-config-google": "0.14.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-import-resolver-typescript": "2.4.0",
+    "eslint-plugin-header": "3.1.1",
+    "eslint-plugin-import": "2.24.2",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-prettier": "4.0.0",
+    "jest": "26.6.3",
+    "jest-junit": "12.2.0",
+    "prettier": "2.3.2",
+    "semantic-release": "17.4.7",
+    "sinon": "11.1.2",
+    "tough-cookie": "4.0.0",
```

Comment:
Given our requirement for this in the session authenticator I think we should move this into `dependencies`

Reply:
Oh wow, I thought I did this, maybe the squash was not worked the way I expected. Thanks!